src/main/java/com/minecolonies/colony/Structures.java

```diff
@@ -32,9 +46,9 @@
     private static       Map<String, List<String>> decorationStyleMap    = new HashMap<>();
 
     /**
-     * Private constructor so Schematics objects can't be made.
+     * Private constructor so Structures objects can't be made.
      */
-    private Schematics()
+    private Structures()
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)